packages/serverless/src/apollo/app/collection.ts

```diff
@@ -30,11 +24,25 @@ export const schemaField = Joi.object({
 
 export const schemaFields = Joi.array().items(schemaField);
 
+export type TCollectionRequest = TDatabaseRequest & {
+  collectionName: string;
+};
+
+export type TCollectionCreateRequest = TCollectionRequest & {
+  schema: TSchemaFieldDefinition[];
+  permission?: number;
+  groupName?: string;
+};
+
+export const CollectionRequest = Joi.object<TCollectionRequest>({
+  collectionName,
+  databaseName,
+});
+
 export const CollectionCreateRequest = Joi.object<TCollectionCreateRequest>({
   collectionName,
   databaseName,
   groupName: groupName.optional(),
-  index: Joi.array().items(collectionIndex.optional()),
```

Comment:
it's no longer allowed to define indexes within collection creation?

Reply:
No it allow to create index within create collection